src/main/scala/edu/ie3/mobsim/model/TripSimulation.scala

```diff
@@ -260,7 +260,7 @@ object TripSimulation extends LazyLogging {
                 s"${ev.getId} drives to a charging hub of type chargingHubHighway."
                 // )
                 makeTripToChargingHub(
-                  "chargingHubHighway",
+                  PoiTypeDictionary("chargingHubHighway"),
```

Comment:
Please use the enum value directly:
```suggestion
                  PoiTypeDictionary.CHARGING_HUB_TOWN,
```